mmv1/templates/terraform/custom_delete/kms_crypto_key_version.erb

```diff
@@ -0,0 +1,10 @@
+    cryptoKeyVersionId, err := parseKmsCryptoKeyVersionId(d.Id(), config)
+	if err != nil {
+		return err
+	}
+    if cryptoKeyVersionId == nil {
+        return nil
+    }
+    userAgent = userAgent + "g"
```

Comment:
this doesn't seem to do anything

```suggestion
```

Reply:
This is more like a dummy line, because in the current custom_delete function, some "userAgent" is parsed in and it's not used by anything. For sure this should be removed.